src/oe-param.js

```diff
@@ -57,7 +115,7 @@ function getInputParameter(index, param, configuration) {
   const parameter = {
     pos: index,
     value: param,
-    type: resolveParameterType(param, configuration)
+    type: resolveParameterType(param, configuration),
```

Comment:
@FlyingWraptor why add a , 🤔

Reply:
Added additional parameter, moved to an if statement, must've forgotten to remove that.